docs/course_materials/compound_data_types.html

```diff
@@ -107,60 +108,33 @@ <h3><b>Popping an element (remove it and return it's value):</b></h3>
                 </section>
                 <section>
                     <h3><b>More actions</b></h3>
-                    <pre><code data-trim data-noescape class="hljs">
-lst[i] = v    # Changes an element or slice by assigning
-lst.extend(l) # Adds items from iterable
-lst.remove(v) # Removes a specific value
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+lst[i] = v      # Changes an element or slice by assigning
+lst.remove(v)   # Removes a specific value
                     </code></pre>
-                </section>
-                <section>
-                    <h3><b>Sequence operations</b></h3>
                     <ul>
-                        <li>All sequence operations are valid on lists:</li>
+                        <li>The .extend() method adds all the elements of an iterable to the end of the list, and returns None:</li>
                     </ul>
-                    <pre><code data-trim data-noescape class="hljs">
-l = ['orange', 'apple', 'strawberry', 'banana']
-print(len(l))     # 4
-                    </code></pre>
-                    <ul>
-                        <li>For loops through the list:</li>
-                    </ul>
-                    <pre><code data-trim data-noescape class="hljs">
-for x in l:
-    print(x)
-
-# orange
-# apple
-# strawberry
-# banana
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+lst = [1,2,3]
+lst.extend([4,5,6])
+print(lst)  # [1,2,3,4,5,6]
```

Comment:
I think it will be better to add here a print(len(l)), so it won't completely be removed

Reply:
I didn't quite understand what you mean :)
Do you mean something like this:
```
print(len(lst)) # 6
```
or
```
lst = [1,2,3]
lst2 = [4,5,6]
lst.extend(lst2)
print(lst)          # [1,2,3,4,5,6]
print(len(lst2)) # 3
# or
print(lst2)        # [4,5,6]
```

Reply:
you've removed the example for print(len(l)).
I just think it should be back.
I think that will do the trick:

lst = [1,2,3]
lst2 = [4,5,6]
lst.extend(lst2)
print(lst)          # [1,2,3,4,5,6]
print(len(lst1)) # 6